.github/workflows/nglint.yml

```diff
@@ -0,0 +1,45 @@
+name: NgLint
+
+on: 
+  pull_request:
```

Comment:
What is the reason behind not adding the GitHub action on push? Wouldn't be better to have it on push too in order to fix whatever the lint is proposing?

Reply:
So as mentioned above, it will run on a push, but only if pushing to an open pull request